pkg/kubernetes/config/kubeconfig.go

```diff
@@ -27,16 +28,12 @@ var DefaultContextName = fmt.Sprintf("%s@%s", DefaultClusterAdminName, DefaultCl
 
 // NewKubeConfig generates a Kubernetes configuration (e.g. for kubectl to use)
 // from the provided machines, and places it in the provided directory.
-func NewKubeConfig(artifactDirectory string, machines []*clusterv1.Machine) (string, error) {
-	master := machine.FirstMaster(machines)
+func NewKubeConfig(artifactDirectory string, machines []*clusterv1.Machine, bl []*baremetalv1.BareMetalMachine) (string, error) {
```

Comment:
Do we create a secret with the new kubeconfig? If we don't we should probably do this so that its available from the management cluster. This is done by the kubeadm controlplane with other providers so if we implement our own control plane then we could do that in there.

Reply:
I believe `wksctl kubeconfig` fetches the secret from `/etc/kubernetes` where kubeadm leaves it.
Would we use this secret for that?  Can't think of another place where we need it.

Reply:
Turns out this function is never used.